string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,29 @@
+/* exported isUpperCased */
+
+/*
+
+  define a function 'isUpperCased' that takes one parameter 'word'
+    define a variable 'upperCaseWordTracker' and assign it to an empty string
+    begin a loop that iterates through 'word'
+      as we iterate through 'word', if the letter at the current index is lower case,
+        return false
+      else if the letter at the index is upper case,
+        add that letter at that index of 'word' to the variable 'upperCaseWordTracker'
+    if 'upperCaseWordTracker' is all uppercase,
+      return true
+
+*/
+
+function isUpperCased(word) {
+  var upperCaseWordTracker = '';
```

Comment:
When using the `.toLowerCase()` or `.toUpperCase()` you don't need to loop over whole string to apply. eg.

`'hello'.toUppercase()` returns `'HELLO'`

so all of this could be reduced to 
```
function isUpperCased(word) {
  return word === word.toUpperCase();
}
```

Reply:
dang i really made that hard on myself huh hahahaha